pages/tutorial/gui.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: page-fullwidth
+title: "Graphical Interface"
+meta_title: "The Alchemist's Graphical Interface"
+subheadline: "Learn the basics!"
+teaser: "Alchemist is equipped with a Java graphical interface, and here is how to use it."
+permalink: "pages/tutorial/gui/"
+comments: true
+#categories:
+#    - design
+header:
+   image_fullwidth: "header_mol.png"
+---
+
+## Starting Alchemist in graphical mode
+
+### Prerequisites
+
+* A working graphical environment
+* A Java 8 (or newer) runtime environment
```

Comment:
Java 8 with OpenJFX 8, Java 11, or a later Java version. Java 9 and Java 10 are not supported, although the Java 8 version of Alchemist might work, if OpenJFX or another JavaFX library is correctly installed..